lib/workers/repository/updates/generate.ts

```diff
@@ -226,7 +226,7 @@ export function generateBranchConfig(
       regEx(/to vv(\d)/),
       'to v$1'
     );
-    if (upgrade.toLowerCase) {
+    if (upgrade.toLowerCase && upgrade.commitMessageLowerCase === 'auto') {
```

Comment:
```suggestion
    if (upgrade.toLowerCase && upgrade.commitMessageLowerCase !== 'never') {
```

@rarkins should we negate to be more save against missing value?